Podfile.lock

```diff
@@ -0,0 +1,28 @@
+PODS:
```

Comment:
Uhm, why do we need to commit `Podfile.lock`? Isn't it specific to the local machine?